tools/telemetry-generator/src/handlers/stageTimingRetriever.ts

```diff
@@ -10,60 +10,74 @@ interface ParsedJob {
 	stageName: string;
 	startTime: number;
 	finishTime: number;
-	totalTime: number;
+	totalSeconds: number;
```

Comment:
Just for clarity within this file. The telemetry event generated in the end still uses `duration`.